### DISCUSSION TITLE
Add system option for theme controller

### DISCUSSION THREAD

**Discussion by RTSAjwad** [saadeghi/daisyui#2887] [Ideas / feature requests]
The current theme controller supports manually picking themes. The default behavior without a theme controller is simply using the system setting. The theme controller could allow for choosing system as an option. That way you can have a theme controller on the page whilst still keeping default functionality if wanted. A drop down theme controller could have:

- System
- Light
- Dark

**Comment by saadeghi** [comment]
https://daisyui.com/components/theme-controller/#theme-controller-using-a-dropdown


"Default" is system

**Reply by RTSAjwad** [reply]
Thank you for the swift reply! I realized myself after posting that mine wasn't behaving as I intended due to the theme being set manually on the <html> tag. Apologies for wasting your time.
